src/App.js

```diff
@@ -1,22 +1,39 @@
-import React from "react";
+import {useState} from "react";
+import {bootcampers} from "../src/bootcampers.js";
 import "./App.css";
 
 function App() {
+
+  const [count, setCount] = useState(0);
+  const [bootcamperIndex, setBootcamperIndex] = useState(Math.floor(Math.random() * bootcampers.length));
+
+  function handleCounterClick() {
+    setCount(count + 1);
+  }
+
+  function handleBootcamperClick() {
+    const randomIndex = Math.floor(Math.random() * bootcampers.length);
```

Comment:
This is great, but again the only minor feedback I have is that this piece of code appears twice!
```
Math.floor(Math.random() * bootcampers.length)
```

It seems like a minor thing, but if we were to refactor this into a function called `getRandomIndex` we could then write unit tests for that function and make sure that the code always works exactly as expected, as well as making it clearer to someone reading the code exactly what our intent is.